CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 ## [Unreleased]
 <details>
   <summary>
-    Changes that have landed in master but are not yet released.
-    Click to see more.
+    * Replace old calendar icon with new one provided by Design.
```

Comment:
This is not how we use the unreleased section.

If you preview the markdown file, you will notice that this change does not make sense.

It uses the summary content to populate a toggleable message that expands and hows the rest of the details content.

The original intent is to uses details to contain a bunch of changes before release. At the moment, we really only have 1 change on deck before a release so we haven't seen it get used much.

Anyways, let's change this to reflect a detail rather than a summary.
Something like:

```
details:
  summary: Changes that had landed in `master` but are note yet released. Click to see more.
  
  * Replace old calendar icon with new one provided by Design.
```